<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicBackgroundSkinDialog</class>
 <widget class="QWidget" name="MusicBackgroundSkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MusicBackgroundSkinDialog{border:1px solid gray}</string>
  </property>
  <widget class="QLabel" name="showPerArea">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>90</y>
     <width>24</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230,115,0);</string>
   </property>
   <property name="text">
    <string>100%






 50%





 0%</string>
   </property>
  </widget>
  <widget class="QSlider" name="bgTransparentSlider">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>95</y>
     <width>20</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>335</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>皮肤与窗口调整</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>453</x>
     <y>2</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mySkin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>我的皮肤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="netSkin">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>推荐皮肤</string>
   </property>
  </widget>
  <widget class="QSlider" name="bgTransparentSliderR">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>95</y>
     <width>20</width>
     <height>200</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>85</y>
     <width>355</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QListWidget" name="themeListWidget"/>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>73</y>
     <width>455</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="customSkin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义皮肤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paletteButton">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>调色板</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>315</y>
     <width>455</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>481</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>showPerArea</zorder>
  <zorder>bgTransparentSlider</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>mySkin</zorder>
  <zorder>netSkin</zorder>
  <zorder>bgTransparentSliderR</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>line</zorder>
  <zorder>customSkin</zorder>
  <zorder>paletteButton</zorder>
  <zorder>line_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
